requirements/install.pip

```diff
@@ -45,5 +45,4 @@ xmltodict==0.10.2
 geojson>=1.3.1
 pymongo==3.4.0
 
-# Lock Jinja version because of changes impacting markdown rendering
-Jinja2<2.9
+Jinja2==2.9
```

Comment:
Can you remove the empty line above and pin it to the real published version ? (2.9.6 right now)